packages/expo-doctor/ts-declarations/metro-config/index.d.ts

```diff
@@ -0,0 +1,230 @@
+declare module 'metro-config' {
```

Comment:
IIRC metro-config ships with types now (?)

Reply:
The `MetroConfig` class is pulled from `resolve-from` in this case, from the target project itself (same as `eas-cli`), so I think I want to do this so I can avoid including all of `metro-config` and also so I can use whatever version is present in the project. Unless there's a way to just include the types from `metro-config`?